C/include/c4Replicator.h

```diff
@@ -342,7 +336,7 @@ extern "C" {
     // TLS options:
     #define kC4ReplicatorOptionRootCerts        "rootCerts"  ///< Trusted root certs (data)
     #define kC4ReplicatorOptionPinnedServerCert "pinnedCert"  ///< Cert or public key (data)
-    #define kC4ReplicatorOptionServerCertVerifyMode "serverVerifyMode" ///< How to verify the server TLS cert (C4ReplicatorServerCertVerificationMode)
+    #define kC4ReplicatorOptionOnlySelfSignedServer "onlySelfSignedServer" ///< Only accept self signed server certs (for P2P, bool)
```

Comment:
Not onlySelfSignedServerCerts to inline with kC4ReplicatorOptionRootCerts or kC4ReplicatorOptionPinnedServerCert?

Reply:
I was trying to avoid making the name comically long....